Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -362,6 +364,13 @@ private void HandlePauseInput(GameTime gameTime)
             // The user wants to resume their play.
             else if (IsPaused && (KeyboardManager.IsUniqueKeyPress(ConfigManager.KeyPause.Value) || KeyboardManager.IsUniqueKeyPress(Keys.Escape)))
             {
+                if (DialogManager.Dialogs.Count != 0)
+                {
+                    DialogManager.Dismiss();
+                    return;
+                }
+
+
```

Comment:
Remove the extra line break here.

Reply:
Actually, we should probably just handle chat here now that I think about it. Using Dismiss() will completely destroy the chat interface.

Use this instead.

```cs
if (ChatManager.IsActive)
    ChatManager.ToggleChatOverlay();
```

It was what you had originally, but I wasn't thinking :crying_cat_face:

Reply:
Aaaaaaaaaaaaaaaaaaaaa what so handle it in PauseScreen.cs where I originally had it? Or keep it in GameplayScreen, because either works

Reply:
Keep it in GameplayScreen